lib/include/jxl/encode.h

```diff
@@ -251,6 +251,24 @@ typedef enum {
    */
   JXL_ENC_OPTION_MODULAR_PREDICTOR = 27,
 
+  /** Fraction of pixels used to learn MA trees as a percentage. -1 = default,
+   * 0 = no MA and fast decode, 50 = the default, 100 = All.
+   */
+  JXL_ENC_OPTION_MODULAR_TREE_PERCENT = 28,
+
+  /** Number of extra (previous-channel) MA tree properties to use. -1 =
+   * default, 0-11 = valid values. Recommended values are in the range 0 to 3,
+   * or 0 to amount of channels minus 1 (including all extra channels, and
+   * excluding color channels when using VarDCT mode). Higher value gives slower
+   * encoding and slower decoding.
+   */
+  JXL_ENC_OPTION_MODULAR_EXTRA_PROPERTIES = 29,
```

Comment:
Maybe `_NB_PREV_CHANNELS` is better. The actual number of extra properties is 4 times this number, since there are 4 extra properties per previous channel.

Reply:
Done